selectel/mks.go

```diff
@@ -84,6 +84,64 @@ func mksClusterV1StateRefreshFunc(
 	}
 }
 
+func mksClusterV1KubeVersionDiffSuppressFunc(k, old, new string, d *schema.ResourceData) bool {
+	if d.Id() != "" {
+		currentMajor, err := kubeVersionToMajor(old)
+		if err != nil {
+			return false
```

Comment:
Can we log those errors in the `[DEBUG]` level? In that case they will be available with `-env TF_LOG=debug` flag.